action.yml

```diff
@@ -12,7 +12,9 @@ runs:
     run: |
       DADJOKE=$(curl -H "Accept: text/plain" https://icanhazdadjoke.com/)
       echo "$DADJOKE"
-      echo "::set-output name=joke::$DADJOKE"
+      echo "joke<<EOF" >> $GITHUB_OUTPUT
```

Comment:
I think I would rather replace lines 15-17 with:

`echo "joke=$DADJOKE" >> $GITHUB_OUTPUT`

What do you think?

Reply:
So I've made some tests and:
- `echo "joke=$DADJOKE" >> $GITHUB_OUTPUT` does not work when joke is multiline
- `echo joke=$DADJOKE >> $GITHUB_OUTPUT` removes new lines from joke (in case of multine sentence)
- proposed solution - works no matters what

- and bonus: current `echo "::set-output name=joke::$DADJOKE"` passes only first line of joke

@mickeygousset considering above - I would stay with what's in PR, let me know what's your opinion : )

Reply:
@mickeygousset sorry for bothering and pinging, but please check above : )

Reply:
@mickeygousset ping ping

Reply:
Ok, let's give it a go.